akka-docs/_sphinx/themes/akka/layout.html

```diff
@@ -219,10 +218,11 @@
     indexName: 'akka_io',
     inputSelector: '#search',
     algoliaOptions: {
-      hitsPerPage: 10,
-      facetFilters: '[' +
-      '["language:' + lang + '","language:general"]' +
-      ',"version:2.4.2"]'
+      hitsPerPage: 5,
+      facetFilters: '[' + '["language:' + lang + '","language:general"]' + ',"version:2.4.2"]',
```

Comment:
Oh, that's a bug I did I think! Glad that we noticed it here, I'll fix on prod and after merging this